### DISCUSSION TITLE
Dose this mean an error issue when getting a timestamp after channel in delev output?

### DISCUSSION THREAD

**Discussion by woshidama323** [go-delve/delve#3201] [Q&A]
I have a process where it get stuck on two goroutines 
Here is the issue description
https://stackoverflow.com/questions/74468601/what-does-this-time-mean-in-delve-goroutine-traceback-info-chan-receive-463432

What does this timestamp means ? why  the other channels without such timestamp works well while two channels with timestamps get stuck ?

**Comment by aarzilli** [comment]
It doesn't mean anything, it was interpreted incorrectly: https://github.com/go-delve/delve/issues/3137
